### DISCUSSION TITLE
Am I able to get direct access to github student benefits, with an email as proof of school attendance?

### DISCUSSION THREAD

**Discussion by GabrielSidhom** [community/community#132800] [GitHub Education]
### Hi everyone,

Am I able to get direct access to github student benefits, with an email as proof of school attendance?

I tried to submit an email screenshot shown down below,
![Screenshot 2024-07-19 15 28 10](https://github.com/user-attachments/assets/7bbbd6c4-0a40-4f3c-8637-a90b0e160bad)
but the AI declined it. I live in the UK, and I don't have a Student ID and I don't get any letters, only emails from the head.

would it be possible to get access to these benefits?

**Comment by SamuelOUS** [comment]
Hi Sploooop,

Unfortunately, it doesn't work if there's no date. I tried with an email and it didn't work. I also tried with my ID, but it didn't have a matriculation date. In that case, I used a screenshot from my college's website that showed my start date in the career, and it worked. So, if your college has a website, you should try that.

If you have any more questions or need further assistance, feel free to ask!
